package.json

```diff
@@ -16,15 +16,19 @@
     "type": "git",
     "url": "git+https://github.com/cfpb/hmda-platform-ui.git"
   },
-  "keywords": ["HMDA"],
+  "keywords": [
+    "HMDA"
+  ],
   "author": "Consumer Financial Protection Bureau",
   "license": "CC0-1.0",
   "bugs": {
     "url": "https://github.com/cfpb/hmda-platform-ui/issues"
   },
   "dependencies": {
     "detect-browser": "1.7.0",
+    "eslint-plugin-prettier": "3.0.0",
```

Comment:
Do we need this dependency?

Reply:
Without it, CRA breaks my local tooling :sweat: